templates/prometheus/kubernetes-nodes.yaml.j2

```diff
@@ -0,0 +1,25 @@
+job_name: 'kubernetes-nodes'
+kubernetes_sd_configs:
+- api_server: https://{{k8s_api_endpoint}}
+  role: node
+  tls_config:
+    insecure_skip_verify: true
+  basic_auth:
+    username: admin
```

Comment:
Do we want to give the admin user creds out? Maybe a view-only user?

Reply:
See previous comments about the separate bug.  I'll take a stab at this but if it starts taking too long, I might have to leave it as a separate work item.